### ISSUE TITLE
Compatibility with Java 1.8

### ISSUE THREAD

**Issue by desposi1** [markathomas/LocationTextField#3]
I cannot reproduce the issue today but I am leaving this here in case anyone else runs into it.

When I first tried displaying the LocationTextField component I was getting a `ClassNotFoundException` for `com.sun.org.apache.bcel.internal.util.Objects.java`. The only thing I found about this class was in the 7u40-b43 JDK. I am running:

```
$ java -version
java version "1.8.0_51"
Java(TM) SE Runtime Environment (build 1.8.0_51-b16)
Java HotSpot(TM) 64-Bit Server VM (build 25.51-b03, mixed mode)
```

I found the [source code](http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/7u40-b43/com/sun/org/apache/bcel/internal/util/Objects.java) and created the file in my projects (DISGUSTING HACKS AT 3AM!) to get everything to work. If I delete the class now, everything still works. Not sure what was happening.

```
/*
* Copyright (c) 2013 Oracle and/or its affiliates. All rights reserved.
* DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
*
* This code is free software; you can redistribute it and/or modify it
* under the terms of the GNU General Public License version 2 only, as
* published by the Free Software Foundation.  Oracle designates this
* particular file as subject to the "Classpath" exception as provided
* by Oracle in the LICENSE file that accompanied this code.
 *
 * This code is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 * version 2 for more details (a copy is included in the LICENSE file that
 * accompanied this code).
 *
 * You should have received a copy of the GNU General Public License version
 * 2 along with this work; if not, write to the Free Software Foundation,
 * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
 *
 * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
 * or visit www.oracle.com if you need additional information or have any
 * questions.
 */
package com.sun.org.apache.bcel.internal.util;

public final class Objects {
    private Objects() {
        throw new IllegalAccessError();
    }

	public static int hashCode(final Object o) {
        return o == null ? 0 : o.hashCode();
    }

	public static boolean equals(Object one, Object two) {
        return one == two || one != null && one.equals(two);
    }
}
```

**Comment by desposi1** [comment]
@markathomas I am closing this issue as solved. Not sure what was happening last night, but it was a minor speed bump. Everything is working as expected today (with no hacks).

**Comment by markathomas** [comment]
this is fixed in the latest version

**Comment by markathomas** [comment]
ahh, one more case.  fixing now.  thanks for reporting

**Comment by desposi1** [comment]
Thanks!

**Comment by markathomas** [comment]
fixed in 2.0.2 - available now

Regards,

Mark Thomas
spatialguru.net@gmail.com
205.529.9013

"Commit to the Lord whatever you do,
    and your plans will succeed." - Proverbs 16:3
